css/pages/card/card.html

```diff
@@ -0,0 +1,283 @@
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dash CSS</title>
+    <link rel="icon" type="image/x-icon" href="../../../resources/logo.png">
+    <link rel="stylesheet" href="../pagesStyle.css">
+    <link rel="stylesheet" href="../../style.css">
+    <script src="https://kit.fontawesome.com/0e6fb8d762.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <nav class="nav_bar">
+        <div class="nav_left_side">
+            <img class="nav_logo" src="../../../resources/logo.png" alt="logo">
+            <span><a class="heading_link" href="google.com">dashcss</a></span>
+        </div>
+        <div class="nav_right_side">
+            <span><a class="heading_link" href="google.com">Docs</a></span>
+            <span><a class="heading_link" href="google.com">Blog</a></span>
+            <span><a class="heading_link" href="google.com"><i class="header_icon link fa fa-github fa-lg">
```

Comment:
href attribute link ?

Reply:
Dummy link.No issue , changed to the right ones for defined ones and the ones which are defined till now I have put #